io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -33,12 +33,12 @@ public function channelReadString(ReadableChannel readableChannel) returns @tain
     }
 }
 
-# Read the entire channel content as a list of lines.
+# Read the entire channel content as a list of lines
```

Comment:
How does this API determine what constitutes a line? It it the new line char?

Reply:
Yes